PKHUD/HUD.swift

```diff
@@ -102,3 +103,30 @@ public final class HUD {
         }
     }
 }
+
```

Comment:
This shouldn't be an UIView extension. It's correct and useful only in Hud.swift